spec/features/displaying_time_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature 'posts are displayed with the time of creation' do
+  scenario 'user can see time next to peep' do
+    sign_in_and_play
+    expect(page).to have_content ("#{Time.new.strftime('%Y-%m-%d')}")
```

Comment:
I found it difficult to write a feature test for posts being displayed with the time of creation. I like that you use expect(page).to have_content Time formatted to Years, Months and Date. Good idea!